balancer-js/src/modules/data/pool/subgraph.ts

```diff
@@ -33,16 +33,16 @@ interface PoolsSubgraphRepositoryOptions {
   query?: GraphQLQuery;
 }
 
-interface SubgraphSubPoolToken extends SubgraphSubPoolTokenFragment {
+export interface SubgraphSubPoolToken extends SubgraphSubPoolTokenFragment {
```

Comment:
I don't think any of these need exported now?

Reply:
You're right - I was exporting them on my previous implementation, but it's not necessary anymore. I'll revert this.